downstream/assemblies/platform/assembly-upgrading-controller.adoc

```diff
@@ -19,14 +19,14 @@ This upgrade procedure assumes that you have a working installation of Ansible a
 
 [NOTE]
 ====
-You can not convert an embedded-database {ControllerName} to a slustered installation as part of an upgrade. Users who want to deploy {ControllerName} in a clustered configuration should back up their {ControllerName} database, install a new clustered configuration on a different VM or physical host, and then restore the database. It is possible to add additional instances later to {ControllerName} if it is already operating on an external database. See link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/clustering.html#ag-clustering[Clustering] in the _Ansible Controller Administration Guide_ for more.
+You can not convert an embedded-database {ControllerName} to a clustered installation as part of an upgrade. Users who want to deploy {ControllerName} in a clustered configuration should back up their {ControllerName} database, install a new clustered configuration on a different VM or physical host, and then restore the database. It is possible to add additional instances later to {ControllerName} if it is already operating on an external database. See link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/clustering.html#ag-clustering[Clustering] in the _Ansible Controller Administration Guide_ for more.
 ====
 
```

Comment:
@emurtoug 

These changes look good to me, but I can't find this assembly included in any title.